spec/version_checker_spec.rb

```diff
@@ -45,8 +53,36 @@ def stub_rubygems_call(version)
   end
 
   def stub_devdocs_call
-    repo = [{ "app_name": "example", "team": "#platform-security-reliability-team" }]
+    repo = [{ "app_name": "example", "team": "#platform-security-reliability-team", "links": { "repo_url": "https://example.com" } }]
     stub_request(:get, "https://docs.publishing.service.gov.uk/gems.json")
       .to_return(status: 200, body: repo.to_json, headers: {})
   end
+
+  def stub_slack_poster_call_with_changes
+    stub_request(:post, "https://slack/webhook")
+    .with(
+      body: { "payload" => "{\"icon_emoji\":\":gem:\",\"username\":\"Gem Release Bot\",\"channel\":\"#platform-security-reliability-team\",\"text\":\"  <https://example.com|example> has unreleased changes since v1.2.2\"}" },
```

Comment:
```suggestion
      body: { "payload" => "{\"icon_emoji\":\":gem:\",\"username\":\"Gem Release Bot\",\"channel\":\"#platform-security-reliability-team\",\"text\":\"<https://example.com|example> has unreleased changes since v1.2.2\"}" },
```